### PR TITLE
Move core game types to a types folder

### PR DESCRIPTION
**Pull request by robertos**
Move some types to their own files.
No functional changes.

### CODE REVIEW

File: client/src/game/types/actions.ts [hunk 1]
```diff
@@ -1,59 +1,18 @@
...
-  type: 'text';
-  text: string;
-}
-
-export interface ActionTurn {
-  type: 'turn';
-  num: number;
-  who: number;
-}
+| ActionClue
```
Comment (robertos) [inline client/src/game/types/actions.ts:5]: now alphabetical :)

File: client/src/websocketInit.ts [hunk 2]
```diff
@@ -161,7 +162,7 @@ const initCommands = () => {
     ) {
       const urlParams = new URLSearchParams(window.location.search);
       const name = urlParams.get('name') || globals.randomName;
-      const variant = urlParams.get('variant') || 'No Variant';
+      const variant = urlParams.get('variant') || DEFAULT_VARIANT_NAME;
```
Comment (robertos) [inline client/src/websocketInit.ts:165]: Introduced a constant to kill 'No Variant' scattered through the code

File: client/src/game/ui/arrows.ts [hunk 3]
```diff
@@ -235,9 +236,13 @@ const animate = (arrow: Arrow, card: HanabiCard, rot: number, giver: number, tur
...
 
-export const click = (event: any, order: ReplayArrowOrder, element: any) => {
+export const click = (
+  event: KonvaEventObject<MouseEvent>,
+  order: ReplayArrowOrder,
+  element: any,
+) => {
   if (
-    event.evt.which === 3 // Right-click
+    event.evt.button === 3 // Right-click
```
Comment (Zamiell) [inline client/src/game/ui/arrows.ts:245]: does this still work the same as button?

Comment (robertos) [reply client/src/game/ui/arrows.ts:245]: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which

The MouseEvent.which read-only property indicates which button was pressed on the mouse to trigger the event. The standard alternatives to this property are MouseEvent.button and MouseEvent.buttons.

Comment (Zamiell) [reply client/src/game/ui/arrows.ts:245]: ok, just makin sure

**Pull request merged**